worlds/rabi_ribi/client/client.py

```diff
@@ -524,7 +556,18 @@ async def rabi_ribi_watcher(ctx: RabiRibiContext):
 
             cur_time = time.time()
 
+            if not ctx.has_zero_health():
+                ctx.has_died = False
```

Comment:
Is the player still at 0 health on game over screen? I thought it got updated back to full health when the continue button popped up, but I might be misremembering.

Reply:
Yeah, I must've looked at a different value; this seems fine.